ngi_reports/stockholm/project_summary.py

```diff
@@ -199,3 +229,55 @@ def get_order_dates(self):
         if self.creation_date:
             dates.append("_Report date:_ {}".format(self.creation_date))
         return ", ".join(dates)
+
+
+    def get_library_method(self):
+        """Get the library construction method and return as formatted string
+        """
+        if self.proj.get('application') == "Finished library":
+            return "Library was prepared by user."
+        try:
+            lib_head = ['Input', 'Type', 'Option', 'Category']
+            lib_meth = re.match(r'(.*) \[[0-9]+\]',self.proj_details['library_construction_method'])
+            if lib_meth:
+                return ("\n".join(["* {}: {}".format(lib_head[c],i.strip()) for c,i in enumerate(lib_meth.group(1).split(',')) \
```

Comment:
Sorry, it is one of my bad habit bro :sweat_smile: , the above will work only one thing is

this
```
lib_meth = self.proj_details['library_construction_method'].rstrip('1234567890')
```
should be
```
lib_meth = re.match(r'(.*) \[[0-9]+\]',self.proj_details['library_construction_method']).group(0)
```

because it will have the document id like [1234] in the end of the string, we just need without it :)